pull_request_template.md

```diff
@@ -1,11 +1,11 @@
 ## Summary
 
-[Change!] Describe your feature, problems you had, notes, improvements and others.
+The signup form was implemented, without any validation and with the functionality of the sign up button, which is to show a console.log() with the information.
 
 ## Screenshots
 
-[Change!] Show the screenshots of the views you modified.
+![demonstration](signup.gif)
 
 ## Trello Card
 
-[Change!] Link to the associated Trello card.
+https://trello.com/c/Bs30bq3X/100-sign-up-i-layout
```

Comment:
do not modify this, this is a template for every PR you are making.
This template works on github to show on pull request creation.
Tthe content of this file should be on github only 😄

Reply:
That's what I realized when I was doing the PR.